Makefile

```diff
@@ -51,3 +51,7 @@ run-local:
 stop-local:
 	docker-compose -f ./docker/local/docker-compose.yml stop
 
+.PHONY: test-e2e
+test-e2e:
+	go build -o polygon-edge main.go
+	env EDGE_BINARY=${PWD}/polygon-edge go test -v ./e2e/... -timeout=30m
```

Comment:
1. Shall we move below `test`?
2. This is a suggestion. Can we prevent `test` command from running e2e test by the following command?

```
go test `go list ./... | grep -v e2e`
```

Reply:
Though not enabled on this PR. What we did in `v3` was to enable `e2e` only when the `E2E_TESTS` env was set to true.

Reply:
1. I like it, moved
2. @ferranbt I found that env var switch a bit tricky, I like @Kourin1996's proposal, applied